dotnet/samples/KernelSyntaxExamples/Example65_HandlebarsPlanner.cs

```diff
@@ -69,9 +89,15 @@ private static async Task RunSampleAsync(string goal, params string[] pluginDire
         }
 
         // The gpt-35-turbo model does not handle loops well in the plans.
-        var allowLoopsInPlan = chatDeploymentName.Contains("gpt-35-turbo", StringComparison.OrdinalIgnoreCase) ? false : true;
+        var allowLoopsInPlan = !chatDeploymentName.Contains("gpt-35-turbo", StringComparison.OrdinalIgnoreCase);
```

Comment:
nit: 
1. Be nice if this explicitly mentions what models to use if loops are desired. Will gpt-4 work better, or it needs to be gpt4-turbo?
2. What is expected to happen if a goal requires loops but the planner is using 35 turbo?

Reply:
It still works w 35-turbo just not as well

Changed comments to:
```
// Use gpt-4 or newer models if you want to test with loops. 
// Older models like gpt-35-turbo are able handle loops but is more prone to syntax errors.
```